siad/network.go

```diff
@@ -65,7 +65,10 @@ func (d *daemon) initializeNetwork(addr string, nobootstrap bool) (err error) {
 		// Every 2 minutes, call CatchUp() on a random peer. This helps with
 		// synchronization.
 		for {
-			go d.CatchUp(d.network.RandomPeer())
+			peer, err := d.network.RandomPeer()
+			if err == nil {
```

Comment:
need to add a comment if you're leaving it `err == nil`

something like 
```go
if err == nil { // only call CatchUp if there was no error
}
```

would be fine